hooks/context_additional_entities.py

```diff
@@ -9,6 +9,18 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 """
+*******************************************************************************
+* DEPRECATION WARNING!                                                        *
+*******************************************************************************
+This hook should not be used if you haven't already implemented it in your 
+pipeline configuration. Support for it is minimal and it will likely be removed 
+in a future release.
```

Comment:
Sounds a little formal.  What do you think of something like

```
This hook should not be used if you aren't already using it.  It probably doesn't
do what you think it does and it will likely be removed in a future release.
```

?